lib/readfile.rb

```diff
@@ -0,0 +1,31 @@
+
+def readFile(file)
+  return File.read(file)
+end
+
+
+def getWords(text)
+  text.gsub!(/[^A-Za-z ]/,'')
+  data = Hash.new
+  temparray = text.to_s.split(" ").map(&:to_s)
+  for word in temparray
```

Comment:
You won't see a lot of "for loops" used in ruby. There's usually a better option available if you sniff around in... 

https://ruby-doc.org/core-2.4.1/Enumerable.html
https://ruby-doc.org/core-2.4.1/Array.html
https://ruby-doc.org/core-2.4.1/Hash.html
https://ruby-doc.org/core-2.4.1/Times.html